resources/views/question.blade.php

```diff
@@ -2,10 +2,45 @@
 
 @section('content')
     <article>
-        <div><h1> Question Title  </h1></div>
-        <div>
-            <div>Vote buttons</div>
-            <div><p>  Question summary </p></div>
+        <div class="py-3"><h2> {{$question->title}}  </h2></div>
+        <div class="row">
+            <div class="col-sm-1">
+                <div class="mb-0"><h1><i class="bi bi-caret-up-fill"></i></h1></div>
+                <div><h1>{{$question->vote}}</h1></div>
+                <div><h1><i class="bi bi-caret-down-fill"></i></h1></div>
+            </div>
+            <div class="col-sm-11"><p> {!! $question->body !!} </p></div>
         </div>
     </article>
+{{--    answers--}}
+    <div class="d-flex flex-column">
+        <div><h4>{{$question->count()}} Answers</h4></div>
```

Comment:
lazy loading count on frontend. Violating MVC and unnecessary performance overhead